macros/clone_schema.sql

```diff
@@ -0,0 +1,266 @@
+{%- macro clone_schema(schema_one, schema_two, comment_tag='') -%}
+
+    {#/* Copies tables, sequences and views from `schema_one` to `schema_two` if `comment_tag` isn't specified. If `comment_tag` argument is specified, it copies only tables and sequences that have `comment` metadata field equal to the passed value of `comment_tag` argument.
+       ARGS:
+         - schema_one (string) : name of first schema.
+         - schema_two (string) : name of second schema.
+         - comment_tag (string) : value of `comment` metadata field that indicates either table or sequence for copying. If it's not specified, all objects from `schema_one` will be copied to `schema_two`.
+       RETURNS: nothing to the call point.
+       SUPPORTS:
+            - Postgres
+            - Snowflake
+    */#}
+
+    {%- if target.type == 'postgres' -%}
+
+        {% set set_pg_function %}
+
+            CREATE OR REPLACE FUNCTION clone_schema(source_schema text, 
+                                                    dest_schema text, 
+                                                    comment_tag text='', 
+                                                    include_recs boolean=true)
+            RETURNS void AS
+
+            $BODY$
+            DECLARE
+            src_oid oid;
+            tbl_oid oid;
+            func_oid oid;
+            object text;
+            description_value text;
+            buffer text;
+            srctbl text;
+            default_ text;
+            column_ text;
+            qry text;
+            dest_qry text;
+            v_def text;
+            count_rows bigint;
+            seqval bigint;
+            sq_last_value bigint;
+            sq_max_value bigint;
+            sq_start_value bigint;
+            sq_increment_by bigint;
+            sq_min_value bigint;
+            sq_cache_value bigint;
+            sq_log_cnt bigint;
+            sq_is_called boolean;
+            sq_is_cycled boolean;
+            sq_cycled char(10);
+
+            BEGIN
+            --1. Create schema.
+            EXECUTE 'DROP SCHEMA IF EXISTS ' || quote_ident(dest_schema) || ' CASCADE; CREATE SCHEMA ' || quote_ident(dest_schema);
+
+            --2. Create sequences.
+            
+            SELECT count(*) INTO count_rows 
```

Comment:
Fetching of sequences counts.

Reply:
@oleg-kolesnikov Could you include this (and the other comments you made on the PR) as inline comments in the code?

Reply:
Seems like have done it.